### DISCUSSION TITLE
Please add Lua binding

### DISCUSSION THREAD

**Discussion by ghost** [dbsoft/dwindows#16] [Ideas]
Lua is a good alternative to Python. The IUP library provides Lua support out of the box:

https://www.tecgraf.puc-rio.br/iup/

**Comment by dbsoft** [comment]
Hmm... not sure about this one I've only heard of LUA used in a few places and I've never personally used it for anything.  Why do you think these bindings would be useful?

**Reply by ghost** [reply]
Lua is more popular than you think. It is embedded into C/C++ programs to provide scripting capacity, e.g: writing plugins for the C/C++ programs without having to write in C/C++ (`mpv` media player uses Lua for this purplose); writing games by scripting in Lua (the game engine is still written in C/C++, but the Lua binding allows developers to write the game in Lua). Lua is not as powerful as Python, though.
